docs-ref-conceptual/index.yml

```diff
@@ -45,34 +45,39 @@ sections:
     style: cards
     className: cardsM
     columns: 3
-    items: 
+    items:
     - image:
         src: https://docs.microsoft.com/azure/media/index/app-service-web.svg
       title: Web Apps
       html: <p>Build, deploy and scale Python web apps on Linux</p>
-      href: https://docs.microsoft.com/en-us/azure/app-service/containers/quickstart-python?toc=%2Fpython%2Fazure%2FTOC.json
-    - image: 
+      href: https://docs.microsoft.com/azure/app-service/containers/quickstart-python?toc=%2Fpython%2Fazure%2FTOC.json
+    - image:
         src: https://docs.microsoft.com/media/logos/logo_kubernetes.svg
       title: Azure Kubernetes Service
       html: <p>Deploy and manage containerized apps</p>
       href: https://docs.microsoft.com/azure/aks/kubernetes-walkthrough?toc=%2Fpython%2Fazure%2FTOC.json
-    - image: 
-        src: https://docs.microsoft.com/en-us/azure/virtual-machines/media/index/virtual-machines.svg
+    - image:
+        src: https://docs.microsoft.com/azure/virtual-machines/media/index/virtual-machines.svg
       title: Virtual Machines
       html: <p>Use a Linux distribution or Windows Server to run your code at scale</p>
       href: https://docs.microsoft.com//azure/virtual-machines/linux/quick-create-cli?toc=%2Fpython%2Fazure%2FTOC.json
+    - image:
+        src: https://docs.microsoft.com/azure/virtual-machines/media/index/AzureFunctions.svg.svg
```

Comment:
.svg.svg might be a problem 😉

Reply:
omgnoooooooo

Reply:
Ugh. Thanks for the catch, @ggailey777. That link was wrong in all sorts of ways. :cry: